subprojects/language-java/language-java.gradle.kts

```diff
@@ -14,6 +14,7 @@ dependencies {
     compile(project(":core"))
     compile(project(":platformJvm"))
     compile(project(":languageJvm"))
+    implementation(project(":snapshots"))
```

Comment:
Should this be `compile` for consistency?

Reply:
I'd leave it like this and instead upgrade to `implementation` everywhere as a separate effort.